librz/bp/bp.c

```diff
@@ -348,13 +381,20 @@ RZ_API int rz_bp_size(RzBreakpoint *bp) {
 	return bpsize;
 }
 
+RZ_API int rz_bp_size_at(RzBreakpoint *bp, ut64 addr) {
```

Comment:
Also doxygen, these functions could be grouped together

Reply:
also RZ_NONNULL, and asserts if they are not there